src/components/EmailVerificationFlow/EmailVerificationRequest/styles.scss

```diff
@@ -0,0 +1,90 @@
+@import '../../../styles/colors.scss';
+
+.divider {
+  width: 100%;
+  height: 1px;
```

Comment:
Style class classNames are global, and most of these are already declared in another file. You could just use the classnames directly, but also here it would be even better to just use the same component since it's quite similar.